apps/docs/components/index.tsx

```diff
@@ -118,6 +119,7 @@ const components = {
       {props.children}
     </Heading>
   ),
+  table: Table,
```

Comment:
nit: do you mind alphabetizing this? I like to keep this file organized because otherwise it's a mess of giant lists